packages/@biomejs/js-api/src/index.ts

```diff
@@ -229,14 +236,35 @@ export class Biome {
 	 */
 	lintContent(
 		content: string,
-		options: LintContentOptions,
-	): PullDiagnosticsResult {
-		return this.withFile(options.filePath, content, (path) => {
-			return this.workspace.pullDiagnostics({
+		{ filePath, fixFileMode }: LintContentOptions,
+	): LintResult {
+		const maybeFixed = fixFileMode
+			? this.withFile(filePath, content, (path) => {
+					let code = content;
+
+					const result = this.workspace.fixFile({
+						path,
+						fix_file_mode: fixFileMode,
+						should_format: false,
+					});
+
+					code = result.code;
+
+					return { content: code };
+			  })
+			: { content };
```

Comment:
Is there a reason why we create a new object, instead of just returning `content`?

Reply:
Fair point, object removed in 4a68be001b2f1264b6ddebf08384016d195ca438

Changelog updated in 871665f97935560889afba9697d6f2921be5cb64